packages/core/contracts/Escrow.sol

```diff
@@ -212,6 +213,8 @@ contract Escrow is IEscrow, ReentrancyGuard {
         }
 
         _safeTransfer(_token, canceler, _amount);
+
+        emit Withdraw(_token, _amount);
```

Comment:
I think the tests should also be modified to check that event has been triggered